src/styles/_layout.scss

```diff
@@ -107,6 +107,14 @@
   &--home {
     padding-bottom: 64px;
     padding-top: 32px;
+
+    .hero__subtext {
```

Comment:
Rather than use multiple lines that won't break down nicely for responsive I like to just put the text in and then dial in a couple max-width breakpoints for it.

Reply:
nice